downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -32,6 +32,58 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ### Improvements
 
+- A bug that resulted in an invalid subtype for concatenation between an xml value and a value of string subtype has been fixed.
+
+```ballerina
+public function main() {
+    xml<xml:Element> x = xml `<bar/>`;
+    string s1 = "foo";
+
+    // used to result in error: incompatible types: expected 'xml<(xml:Element|xml:Text)>', found 'xml'
+    xml<xml:Element|xml:Text> r1 = x + s1;
+
+    "foo"|"bar" s2 = "foo";
+    // used to emit no error for this
+    xml<xml:Element|xml:Comment> r2 = x + s2;
+}
+```
+
+- A bug that resulted in a compile time error for attribute access of an xml variable of non `xml:Element` type singleton value and the union of xml singleton types have been fixed.
```

Comment:
This is not a breaking change, right?

Reply:
We can skip this if so.

Reply:
Added this under the `Improvements` section can't this be considered as an improvement

Reply:
We can just remove it. Since it is a bug fix. 

We only list backward-incompatible bug fixes here.